src/utilities/outputParsers.ts

```diff
@@ -0,0 +1,18 @@
+import { setOutput } from '@actions/core';
+import { Result } from 'semantic-release';
+
+enum OutputParameters {
+  ReleasedVersion = 'version',
+  ReleasedType = 'type',
+}
+
+export const reportResults = (result: Result): void => {
+  if (result === false) {
+    throw new Error('No new version has been released.');
+  }
```

Comment:
I would move this in `src/index.ts` and set a failed status instead of throwing an error. The fact that no new release has happened is not really an error. In my opinion...

Reply:
This can't be an error, this has to be an output. Releases themselves trigger a new flow, which will naturally result in this behavior. An output is then needed to skip `npm publish` step.

Reply:
Provided that GitHub Actions does not offer any other exist statuses beyond success and failed, I think that failing the action in the event of no release being pushed is a natural expectation.
In the exceptional case that you would want to continue the workflow, you can still do that via `if: always()` or `if: failed()`.